src/Hook/useSignUp.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export const useSignUp = async () => {
+  try {
+    const response = await axios.post(`/api/auth/signup`, {
+      firstName: "Dhruv",
+      lastName: "Samant",
+      email: "samantdhruv@gmail.com",
+      password: "9130609806",
+    });
+    // saving the encodedToken in the localStorage
+    localStorage.setItem("token", response.data.encodedToken);
+  } catch (error) {
+    console.log(error);
+  }
+};
```

Comment:
appreciation - nice work that you have added a custom hook for signup

Reply:
thanks!